src/content/release/breaking-changes/system_context_menu_controller_show.md

```diff
@@ -0,0 +1,117 @@
+---
+title: SystemContextMenuController.show Deprecated
+description: >
+  `SystemContextMenuController`'s `show` method is deprecated and replaced by
+  its `showWithItems` method.
+---
+
+## Summary
+
+`SystemContextMenuController.show` is deprecated. The same functionality can be
+achieved by passing `SystemContextMenu.getDefaultItems` to
+`SystemContextMenuController.showWithItems`.
+
+## Background
+
+The iOS-drawn `SystemContextMenu` feature was originally added without the
+ability to control which items are shown in the menu. The platform would decide
+which items to show based on the current input connection.
```

Comment:
Just curious, what's the "current input connection"?

Reply:
I'll link the docs, it's probably not obvious to most readers actually. It means the current field that the user is focused on (TextInputConnection).